easybuild/easyconfigs/b/binutils/binutils-2.36.1.eb

```diff
@@ -0,0 +1,22 @@
+name = 'binutils'
+version = '2.36.1'
+
+homepage = 'https://directory.fsf.org/project/binutils/'
+
+description = "binutils: GNU binary utilities"
+
+toolchain = SYSTEM
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['e68edeaaeb6ca9687b6dcbaedd1b376506baad2d48de26a885fc5ab6acb839da']
+
+builddependencies = [
+    ('flex', '2.6.4'),
+    ('Bison', '3.7.6'),
+    # zlib required, but being linked in statically, so not a runtime dep
+    ('zlib', '1.2.11'),
+    ('makeinfo', '6.7'),
```

Comment:
Not needed, bypass with
buildopts = 'MAKEINFO=true'
installopts = buildopts
like in binutils-2.36.1-GCCcore-10.3.0.eb

And then drop the makeinfo-6.7.eb

Reply:
For reference from slack discussion: makeinfo needs to depend on Perl, and, pulling in Perl as a system level dependency as well would really hurt.

Reply:
I added the build dep to avoid the use of `MAKEINFO=true`. Which one is better?

Reply:
Due to https://github.com/easybuilders/easybuild-easyconfigs/issues/12067 using MAKEINFO=true (or should it be /bin/true?) is preferable here.